src/main/java/com/maddyhome/idea/vim/ui/ex/ExEntryPanelService.kt

```diff
@@ -95,6 +95,18 @@ public class ExEntryPanelService : VimCommandLineService {
     return panel
   }
 
+  public override fun createWithoutShortcuts(
```

Comment:
`createSomething` usually used when we actually create a new instance of the object.
Here we reuse initialized instance of the ExEntryPanel, so I'd say that this method should be named also `getInstance....`

Reply:
In IdeaVim, we reuse the initialized instance, but this is unlikely to be the case for Fleet. The implementation details of IdeaVim should not influence naming of the new API
Previously, we used `getInstanceWithoutShortcuts()` and then called `activate()` on it. Now, however, we have adopted a more intuitive approach by directly calling `createWithoutShortcuts()`